### PR TITLE
Aditional information for users building from source code

### PR DESCRIPTION
**Pull request by igorcosta**
Pullrequest to the correct branch.

### CODE REVIEW

**Comment by srowen** [comment]
Nah, I think this is purposely covered in more detail, separately, at http://spark.apache.org/docs/latest/building-spark.html which this generic overview page links to. I don't think we want to push duplicate info back into this wiki.

**Comment by igorcosta** [comment]
Sort of I agree, but if you're just getting started you need other options specially with hive support right out of the box. I took myself as example.

**Comment by srowen** [comment]
Rather than duplicate the info, I'd suggest just adding to the following sentence to highlight that you can find info about how to build with, say Hive support at the following link ...

**Comment by rxin** [comment]
@srowen I agree if this adds a lot of lines, but since this is only adding one line, and a lot of people will probably want the Hive support, seems no brainer to me.

File: README.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ To build Spark and its example programs, run:
 
     mvn -DskipTests clean package
 
+To build Spark with support for Hadoop, Hive and Python shell for example, you could use
+
+     mvn -Pyarn -Phive -Phive-0.13.1 -Phive-thriftserver -Phadoop-2.4 -Dhadoop.version=2.4.0 -DskipTests clean package
```
Comment (rxin) [inline README.md]: do we need both -Phive and -Phive-0.13.1?

**Comment by srowen** [comment]
OK I am convinced, merge it. I think both hive profiles are needed in this example?

**Comment by igorcosta** [comment]
@srowen Just -Phive . Ive just added exceeded.

**Comment by srowen** [comment]
@igorcosta are you saying you need to make an update or that this should be ready as is?

**Comment by igorcosta** [comment]
@srowen ignore it. ready as is.

**Comment by srowen** [comment]
@igorcosta sorry a few more trivial things -- what about the build command is needed to use PySpark? you could say "Hadoop 2.4" rather than "Hadoop" since that's specifically what the build is configured for, and likewise Hive 0.13. Finally I think you can show this command as a line of code with the triple-backticks. Might make it look nicer on the home page.

**Comment by igorcosta** [comment]
@srowen I agree.

**Comment by srowen** [comment]
@igorcosta are you going to update this PR accordingly or do you want me to edit on merge? it would be almost like making a new PR so I wanted to check first.

**Comment by igorcosta** [comment]
Yep will create a new PR

**Comment by srowen** [comment]
You can just push to this one.

**Comment by srowen** [comment]
@igorcosta would you mind updating this PR accordingly, or else would you mind closing this PR?

**Pull request closed without merging**